packages/virtualdom/src/index.ts

```diff
@@ -747,12 +747,34 @@ class VirtualElement {
   }
 }
 
+export
```

Comment:
Class needs a docstring.

Reply:
Done. I added docstrings to all the new stuff in `virtualdom`